core/src/main/resources/jenkins/security/ApiTokenProperty/help-tokenStore.html

```diff
@@ -0,0 +1,12 @@
+<div>
+    API tokens offer a way to make authenticated CLI or REST API calls.
+    See <a href="https://jenkins.io/redirect/api-token">our wiki</a> for more details.<br/>
+    The username associated with each token is your Jenkins username.<br/>
+    <br/>
+    Some good practices for keeping your API tokens secure are:
+    <ul>
+        <li>Use a different token for each application so that if an application is compromised you can revoke its token individually.</li>
+        <li>Regenerate the tokens every 6 months (depending on your context). We put an indicator concerning the age of the token.</li>
```

Comment:
Maybe "put" -> "display". It's a little hard to figure out where this indicator is being put.